### ISSUE TITLE
Would be great to capture the participated view from native notification view

### ISSUE THREAD

**Issue by ijm8710** [GitHawkApp/GitHawk#2613]
Not sure this is captured in the newish inbox menu redesign. Great job again on that by the way. My vision would be for it to basically include any item you follow watch comment or author.

**Comment by Huddie** [comment]
2 options:
1. You want to split up notification into (All, participating, etc) Meaning once you click the issue it is read and disappears.
2. You want to have a section that shows all issues you're currently watching, commented on, or authored. In which case, we had that in inbox redesign (https://github.com/GitHawkApp/GitHawk/issues/2413#issuecomment-443692746, subscribed) but removed it

**Comment by ijm8710** [comment]
Hey @Huddie 

Two comments

1. [so “participating” is one of the desktop notification views and we don’t currently capture it](https://i.imgur.com/a6NKeP2.jpg)
2. I had a brain-fart. What I would really love is to have a view of items you’ve commented and/or authored. Essentially anything where you’ve directly provided feedback in. Ideally, this differs from 1) as I’d love for it to be inclusive of already read items as well

Originally, I mistakenly included follow and watch in my post, but that was unintentional. I know we didn’t see eye-to-eye when discussed previously but I still feel an unread “subscribed” alert is the same as an unread “inbox” alert-that we already have. And if you want unread as well, that’s just “all”.

**Comment by Huddie** [comment]
@ijm8710 I'd suggest looking to see if Github documentation has a filter for this or if its a combination of some filters. Otherwise, it requires client-side filtering of the "all" category.

**Comment by ijm8710** [comment]
@Huddie I think it does [here](https://developer.github.com/v3/activity/notifications/)

**Comment by Huddie** [comment]
That's under notifications. notification become unread and disappear. You won't get read and unread, unfortunately.

**Comment by ijm8710** [comment]
Good point. For comparison, codehub has a profiled events section that represents this. They have the participating notification view as well for that other piece. I’ll do a little more digging for the first, but haven’t found anything yet.
